src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/EmailPreview/EmailPreview.tsx

```diff
@@ -6,25 +6,22 @@ import { themeSpacing } from '@amsterdam/asc-ui'
 import FormFooter from 'components/FormFooter'
 import ModalHeader from '../ModalHeader/ModalHeader'
 
+const formFooterHeight = 16
```

Comment:
This is not correct; the units used in `themeSpacing` are four pixels each. This means that the height of the footer is 64 pixels and not 16. Also, this constant shouldn't be in this component, but should live in the context of the footer component and the footer component should use that value.

Reply:
16 x 4 = 64

Reply:
ok